_posts/2021-02-05-solidity-fuzzer-update.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+published: true
+title: 'Solidity Fuzzer Update'
+date: '2021-02-05'
+author: Bhargava Shastry
+category: Explainers
+---
+
+Security vulnerabilities and bugs detract from software quality.
+To discover them early, we have adopted fuzz testing: feeding randomly generated programs to the Solidity compiler and observing the compilation runtime and code generated.
+Since Q1 2019, the Solidity compiler is fuzz tested via Google's [open-source software fuzz][1] (oss-fuzz) framework.
+In this post, we briefly describe the work that has been done on this front, and work that is currently in progress.
+
+### Fuzzing Overview
+
+Broadly speaking, we have developed two classes of fuzzers:
+  - Front-end fuzzers that test the compiler front-end (parser/analyser).
+  - Back-end fuzzers that test the compiler back-end (optimiser).
+
```

Comment:
Here the spacing looks super tight in the preview but I don't know if there's a way we can fix it.

Reply:
Ah right, thanks for the preview link. I will try to reformat the text a little bit more so it looks a little more tidy. I suppose one thing that can be done is to increase heading size since we don't have numbered headers i.e., `Section 1` and `Section 1.1` can have the same heading size making them look better formatted although `1.1` is "below" `1`